2024.html

```diff
@@ -71,6 +71,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://villapirorum.netlify.app/">Antoine Villepreux</a></li>
             <li><a href="https://cssence.com/">Matthias Zöchling</a></li>
             <li><a href="https://github.com/css-naked-day/css-naked-day.github.io/edit/master/2024.html"><strong>Yours?</strong></a></li>
+            <li><a href="https://robin.is">Robin Massart</a></li>
```

Comment:
Please sort by last name (see other entries), and add a leading slash (similar to other URLs).